src/Components/Timer/Timer.js

```diff
@@ -0,0 +1,23 @@
+import { View, Text, Button } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+
+import styles from './Timer.style';
+
+function Timer() {
+    const navigation = useNavigation();
+    const onPressHandler = () => {
+        navigation.navigate('TimerScreen');
+    }
+
+    return (
+        <View style={styles.rootContainer}>
+            <Text>TIMER</Text>
+            <Button
```

Comment:
Doporučuji si vytvořit custom button, protože s tímhle se pak na hovno pracuje :)